core/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.jenkins.plugin-modernizer</groupId>
+        <artifactId>plugin-modernizer-parent-pom</artifactId>
+        <version>${revision}${changelist}</version>
```

Comment:
If we are using CD also for the tool (Which I like very much) I think we are missing some files like https://www.jenkins.io/doc/developer/publishing/releasing-cd/#update-maven-pom-and-config

Also incrementals must be enabled (I would suggest to check some plugin with CD enabled)

Reply:
Will take a look into it

Reply:
Hey! @jonesbusy I have enabled incrementals and added Dependabot. Will configure others later.